common/lib/container-definitions/src/loader.ts

```diff
@@ -271,32 +263,14 @@ export interface IContainer extends IEventProvider<IContainerEvents>, IFluidRout
      */
     disconnect(): void;
 
-    /**
-     * Dictates whether or not the current container will automatically attempt to reconnect to the delta stream
-     * after receiving a disconnect event
-     * @param reconnect - Boolean indicating if reconnect should automatically occur
-     * @deprecated - 0.58, This API will be removed in 1.0
-     * Use `connect()` and `disconnect()` instead of `setAutoReconnect(true)` and `setAutoReconnect(false)` respectively
-     * See https://github.com/microsoft/FluidFramework/issues/9167 for context
-     */
-    setAutoReconnect?(reconnect: boolean): void;
-
-    /**
-     * Have the container attempt to resume processing ops
-     * @deprecated - 0.58, This API will be removed in 1.0
-     * Use `connect()` instead
-     * See https://github.com/microsoft/FluidFramework/issues/9167 for context
-     */
-    resume?(): void;
-
     /**
      * The audience information for all clients currently associated with the document in the current session
      */
     readonly audience: IAudience;
 
     /**
      * The server provided ID of the client.
-     * Set once this.connected is true, otherwise undefined
+     * Set once this.connectionState === ConnectionState.Connected is true, otherwise undefined
      * @alpha
```

Comment:
What does alpha mean exactly? When will this graduate to not have this tag?

Reply:
Looks like the alpha tag indicates that there is still active consideration if the API should exist on `IContainer` (see https://github.com/microsoft/FluidFramework/pull/8328). I think at this point it is safe to remove the alpha tag since we are very firm on keeping this API.

Reply:
The alpha tags here were for items that really shouldn't be on `IContainer`, but were getting usage on `Container`, so to get customers to move to using `IContainer` they were added here.  Ideally they should be removed, but require more substantial engineering effort to eliminate.

We should leave the alpha tags on for their original intention - to help steer new developers away from depending on them, since they should eventually be removed.  It would probably be good to add a comment here though explaining that status.